ispdb/educa.madrid.org

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<clientConfig version="1.1">
+    <emailProvider id="educa.madrid.org">
+        <domain>educa.madrid.org</domain>
+
+        <displayname>Plataforma EducaMadrid</displayname>
+        <displayShortName>EducaMadrid</displayShortName>
+
+        <incomingServer type="imap">
+            <hostname>imap.educa.madrid.org</hostname>
+            <port>993</port>
+            <socketType>SSL</socketType>
+            <authentication>password-cleartext</authentication>
+            <username>%EMAILLOCALPART%</username>
+        </incomingServer>
+        <incomingServer type="pop3">
+            <hostname>pop.educa.madrid.org</hostname>
+            <port>995</port>
+            <socketType>SSL</socketType>
+            <authentication>password-cleartext</authentication>
+            <username>%EMAILLOCALPART%</username>
+        </incomingServer>
+        <outgoingServer type="smtp">
+            <hostname>smtp01.educa.madrid.org</hostname>
+            <port>587</port>
+            <socketType>STARTTLS</socketType>
+            <authentication>password-cleartext</authentication>
+            <username>%EMAILLOCALPART%</username>
+        </outgoingServer>
+        <outgoingServer type="smtp">
+            <hostname>smtp01.educa.madrid.org</hostname>
+            <port>465</port>
+            <socketType>SSL</socketType>
```

Comment:
Please use SSL port 465 as first and preferred config. Just change the order, that's all.

Reply:
Otherwise, the config looks fine. (I haven't actually tried the config file.) r+ with this change.

Reply:
Otherwise, the config looks fine. r+ with this change,

Reply:
Done.